.ecp-gitlab-ci.yml

```diff
@@ -49,62 +54,75 @@ stages:
   dependencies:
    - BuildKokkos
 
-BuildKokkos Serial:
-  variables:
-    BACKENDS: "Serial"
-  extends: .BuildKokkos
-
-Build Serial:
-  variables:
-    BACKENDS: "Serial"
-  extends: .Build
-  dependencies:
-   - BuildKokkos Serial
-
-BuildKokkos OpenMP:
-  variables:
-    BACKENDS: "OpenMP"
-  extends: .BuildKokkos
-
-Build OpenMP:
-  variables:
-    BACKENDS: "OpenMP"
-  extends: .Build
-  dependencies:
-   - BuildKokkos OpenMP
+#BuildKokkos Serial:
```

Comment:
Should we remove the commented code?

Reply:
This is still work in progress, ultimately these will get reenabled!